exchange/src/client_manager/client_manager.hpp

```diff
@@ -30,7 +31,8 @@ class ClientManager {
 
 public:
     void add_client(
-        const std::string& uid, float capital = STARTING_CAPITAL, bool active = false
+        const std::string& uid, float capital = STARTING_CAPITAL,
+        bool is_local_algo = false, bool active = false
```

Comment:
Don't do default arguments. What if I want to specify `is_local_algo` but not `capital`?

Just use function overloading.

Reply:
Good point